### ISSUE TITLE
should exit counts capture exit reasons?

### ISSUE THREAD

**Issue by jwdeitch** [digitalocean/vmtop#13]
particularly interested in capturing the number of times the exit reason is stated as `EXCEPTION_OR_NMI` ? 👼 
any interest in this?

**Comment by jdesfossez** [comment]
Yes definitely, and it would be easy to add in the current ebpf script, the issue we have is reporting, there are 60+ exit reasons IIRC and I don't think we want one column for each (either on the console or the CSV).
For the CSV I think we could maybe dump a dict-like in a field like: `{HLT: 1; NMI: 42, ...}` do you have another idea ?
And on the console, I just don't know :-)

**Comment by jwdeitch** [comment]
hmm good q - I'll try the csv dump (I agree w/ the dict-like structure). not sure either about prometheus or console :) - I will give it some thought

**Comment by jdesfossez** [comment]
Prometheus will be fine because the exit reason can be a label so it won't show up unless it's used, the console is my biggest concern.
The other tools that present exit reasons live on the console (`kvm_stat`, `perf kvm`) usually only work with a filter for specific VMs or show the sum from all the VMs.
But I am fine with starting with the CSV/prometheus only and just the total on the console.
